plugins/org.python.pydev.core/build.properties

```diff
@@ -1,4 +1,5 @@
 source.core.jar = src/,\
+                  tests/,\
```

Comment:
Interesting. Is this how they remove the test code? I thought they use a script?

Reply:
Looks like they use a script to add the tests back.

Reply:
Correct